_components/utilities/layout-grid.md

```diff
@@ -4,350 +4,504 @@ layout: styleguide
+</div>
+{% endcapture %}
+{{ grid-wrapping }}
+</div>
+
+<div class="usa-accordion-bordered usa-code-sample margin-top-4">
+  <button class="usa-accordion-button" aria-controls="code-wrapping" aria-expanded="true">Code</button>
+  <div id="code-wrapping" class="usa-accordion-content">
+<div markdown="1">
+```html
+{{ grid-wrapping | strip }}
+```
+</div>
+  </div>
+</div>
+
+<h2 id="gutters">Gutters</h2>
+
+<h3>Default gutter</h3>
+<p class="font-sans-6 measure-3"><code class="docs-inline-code">.grid-gap</code> adds a grid-gap between grid-columns in the grid-row, to a value set as <code class="docs-inline-code">$theme-column-gap</code> in settings (16px default).</p>
+{{ grid-markers }}
+
+<div class="docs-grid-example">
+{% capture grid-gutters %}
+<div class="grid-row grid-gap">
+  <div class="grid-col-4">
+    <div>.grid-col-4</div>
+  </div>
+  <div class="grid-col-4">
+    <div>.grid-col-4</div>
+  </div>
+  <div class="grid-col-4">
+    <div>.grid-col-4</div>
+  </div>
+</div>
+{% endcapture %}
+{{ grid-gutters }}
+</div>
+
+<div class="usa-accordion-bordered usa-code-sample margin-top-4">
+  <button class="usa-accordion-button" aria-controls="code-gutters" aria-expanded="true">Code</button>
+  <div id="code-gutters" class="usa-accordion-content">
+<div markdown="1">
+```html
+{{ grid-gutters | strip }}
+```
+</div>
+  </div>
+</div>
+
+<h3>Large gutter</h3>
+<p class="font-sans-6 measure-3"><code class="docs-inline-code">.grid-gap-lg</code> adds a grid-gap between grid-columns in the grid-row, to a value set as <code class="docs-inline-code">$theme-column-gap-large</code> in settings (32px default).</p>
+{{ grid-markers }}
+
+<div class="docs-grid-example">
+{% capture grid-gutters-lg %}
+<div class="grid-row grid-gap-lg">
+  <div class="grid-col-4">
+    <div>.grid-col-4</div>
+  </div>
+  <div class="grid-col-4">
+    <div>.grid-col-4</div>
+  </div>
+  <div class="grid-col-4">
+    <div>.grid-col-4</div>
+  </div>
+</div>
+{% endcapture %}
+{{ grid-gutters-lg }}
+</div>
+
+<div class="usa-accordion-bordered usa-code-sample margin-top-4">
+  <button class="usa-accordion-button" aria-controls="code-gutters-lg" aria-expanded="true">Code</button>
+  <div id="code-gutters-lg" class="usa-accordion-content">
+<div markdown="1">
+```html
+{{ grid-gutters-lg | strip }}
+```
+</div>
+  </div>
+</div>
+
+<div markdown="1" class="font-sans-6 measure-3">
+<h2 id="sass-mixins">Sass mixins</h2>
+When using the source Sass files, you have the option of using Sass variables and mixins to create custom, semantic, and responsive page layouts. Our predefined grid classes use these same variables and mixins to provide a whole suite of ready-to-use classes for fast responsive layouts.
+
+### Variables
+Variables and maps determine the number of columns, the gutter width, and the media query point at which to begin floating columns. We use these to generate the predefined grid classes documented above, as well as for the custom mixins listed below.
```

Comment:
The only theme variables we _intend_ folks to change wrt grids are the gap widths. The container default with is not currently set as a variable (but it could be).

While it may be technically _possible_ to change the value of breakpoint names and widths, I don't think we should encourage it. These values are not set to `!default` and aren't intended to be changed or overridden.

Users can (and should!) set which breakpoints they output, however.

Reply:
>The only theme variables we intend folks to change wrt grids are the gap widths. The container default with is not currently set as a variable (but it could be).

We can remove it for now for clarity, but I suspect people practicing content-first responsive design may need to adjust breakpoints based on their content.

Reply:
Removed in 675d02c.

Reply:
I'm sympathetic to content-first responsive design, but as a system I think we have to move in the direction of more fixed breakpoints to help assure that components work as expected from project to project. It's the poison I pick, I guess.